db/migrate/20190908183511_set_backordered_inventory_to_on_hand.rb

```diff
@@ -0,0 +1,5 @@
+class SetBackorderedInventoryToOnHand < ActiveRecord::Migration
+  def up
+    Spree::InventoryUnit.backordered.update_all(state: 'on_hand')
```

Comment:
It's better to avoid existing models. Either use plain SQL or define your own model here. Otherwise this will fail when we delete the `backordered` scope from InventoryUnit. I think you can just include this in the migration:

```rb
class SpreeInventoryUnit < ActiveRecord::Base
  scope :backordered, -> { where state: 'backordered' }
end
```
But plain SQL might be easier here.

Reply:
Yes, please. It's very painful when later on the model changed what the migration was relying on 🙈

Might be time to introduce https://github.com/testdouble/good-migrations

Reply:
yes, I think that's correct Kristina.

Reply:
@mkllnk I've updated the migration to plain SQL.